### PR TITLE
hw-mgmt: scripts cleanup optimization

### PR DESCRIPTION
**Pull request by sholeksandr**
scripts cleanup and optimization

Signed-off-by: Oleksandr Shamray <oleksandrs@nvidia.com>

### CODE REVIEW

**Comment by sholeksandr** [comment]
RERUN

**Comment by sholeksandr** [comment]
RERUN

File: usr/usr/bin/hw-management-thermal-events.sh [hunk 1]
```diff
@@ -417,53 +405,41 @@ if [ "$1" == "add" ]; then
...
-		for ((i=1; i<=max_lcs; i+=1)); do
 			if [ -f "$3""$4"/lc"$i"_powered ]; then
 				ln -sf "$3""$4"/lc"$i"_powered $system_path/lc"$i"_powered
 				event=$(< $system_path/lc"$i"_powered)
 				if [ "$event" -eq 1 ]; then
 					echo 1 > $events_path/lc"$i"_powered
 				fi
 			fi
-		done
-		for ((i=1; i<=max_lcs; i+=1)); do
```
Comment (vadimp-nvidia) [inline usr/usr/bin/hw-management-thermal-events.sh:430]: I understand you want to have one loop for all lc{i}_xxx
for ((i=1; i<=max_lcs; i+=1)); do
...
done
which seems logical, diff looks like for loop is removed at all.
AM I missing something in fiff?

Comment (vadimp-nvidia) [reply usr/usr/bin/hw-management-thermal-events.sh:430]: OK. I see, we have first loop: for ((i=1; i<=max_lcs; i+=1)); do

**Review by vadimp-nvidia — APPROVED** [review]

**Pull request merged**